docs/spec.md

```diff
@@ -5570,6 +5565,89 @@ Produces a `result` tuple from values `val`.
 // %result: ([1.0, 2.0], (3))
 ```
 
+### uniform_dequantize
+
+#### Semantics
+
+Performs element-wise conversion of uniform quantized tensor `operand` to a
+floating point tensor `result` according to the quantization parameters defined
+by the `operand` type.
+
+Formally, `result = (operand - zero_point(operand)) * scale(operand)`.
```

Comment:
Btw, I don't find the definition of `scale(...)` and `zero_point(...)` in the Types section.  Do these function implicitly handle the `quantization_dimension`?

Also, please add a per-axis example below.

Reply:
>> Btw, I don't find the definition of scale(...) and zero_point(...) in the Types sectiIn some parts of the spec document we are informally using some macros like `scale(...), zero_point(...), quantization_dimension(...)`. This treatment is similar to the usages of other macros like like `element_type(...), rank(...), shape(...)`.    We have a [ticket](https://github.com/openxla/stablehlo/issues/289) to augment the  specification doc with the details of all of these.  I have added an action item to address the quantization related macros  https://github.com/openxla/stablehlo/issues/289#issuecomment-1559823158.


>> Also, please add a per-axis example below.

Done.

I note that, as per the [guideline](https://github.com/openxla/stablehlo/blob/main/docs/spec_checklist.md#stablehlo-specification-checklist), Item 6 , we only use one example per op, irrespective of the type ([AddOp](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#add) as an example). In future we have plans to explore further into this and, if needed, improve the situation. 

I believe the exception in this case makes sense because in the spec we do not have any example for per-axis quantization.